pages.id/rubocop.md

```diff
@@ -0,0 +1,32 @@
+# rubocop
+
+> Menganalisa file Ruby.
+> Informasi lebih lanjut: <https://docs.rubocop.org/rubocop/usage/basic_usage.html>.
+
+- Memeriksa semua file dalam direktori saat ini (termasuk direktori-direktori di dalamnya):
+
+`rubocop`
+
+- Memeriksa satu atau lebih file atau direktori secara khusus:
+
+`rubocop {{lokasi/ke/file}} {{lokasi/ke/direktori}}`
```

Comment:
```suggestion
`rubocop {{jalan/menujufile}} {{jalan/menuju/direktori}}`
```
[Reference](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

Reply:
Thanks for the suggestions. Is `jalan` widely used for Indonesian translation across literatures?

`jalan` reads unfamiliar to me. However, I'm okay if it's widely used in this repo.

Reply:
> Thanks for the suggestions. Is `jalan` widely used for Indonesian translation across literatures?
> 
> `jalan` reads unfamiliar to me. However, I'm okay if it's widely used in this repo.

Yes it is widely used across all Indonesian pages.